src/pages/instances/actions/DeleteInstanceBtn.tsx

```diff
@@ -39,7 +51,13 @@ const DeleteInstanceBtn: FC<Props> = ({ instance }) => {
       isLoading={isLoading}
       icon="delete"
       title="Confirm delete"
-      confirmationMessage={`Are you sure you want to delete instance "${instance.name}"? This action cannot be undone, and can result in data loss.`}
+      confirmationMessage={
+        <>
+          Are you sure you want to delete instance{" "}
+          <InstanceName instance={instance} />? This action cannot be undone,
```

Comment:
I didn't recognize - we seemed to have had the quotes around the name sometimes, but sometimes not. I think it makes sense to have them, when it is not a link.

Reply:
Good catch! Yes, let's have a rule - when we have just created a new item and added it to the list - it should be a link (instance, profile creation). For snapshot and project creation it should be a regular message.
I do agree we need to make it more obvious. I would suggest bolding the item's name. It will pop up even more. If bolded, we don't really need the quotation marks. Can we try this?